docs/sections/footer.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+  <script src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+  <script src="/cocoandeve-styleguides/js/main.js"></script>
+</head>
+
+<body class="">
+  <div class="container mt-4">
+    <h1>Footer</h1>
+  </div>
+  <footer class="pt-md-6 pb-md-6 pt-2 pt-lg-4 pb-1 mobile-wrapper">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-5">
+          <h2>Newsletter</h2>
```

Comment:
in the component you used the H1 here the H2, I'd use a H5 with the proper class to match the font-size